hackerank/js-practice/angry-professor.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+// Complete the angryProfessor function below.
+function angryProfessor(studentsThreshold, arrivalTime) {
+  const arrivedStudents = arrivalTime.filter((studentTime) => studentTime <= 0);
+  if (arrivedStudents.length < studentsThreshold) {
+    return "YES";
```

Comment:
I recommend extracting `"YES"` and `"NO"` to a constant, and we can reuse late in code.

Reply:
I agree 👍🏻